integration-tests/kubernetes/utils.go

```diff
@@ -17,3 +20,28 @@ func CheckPodsRunning(pods v1.PodList, expectedPods int) bool {
 	}
 	return true
 }
+
+// CheckForOpenebsNamespace returns true if openebs namespace exists in v1.NamespaceList
+func CheckForOpenebsNamespace(namespaces v1.NamespaceList) bool {
```

Comment:
Set a TODO here.
move this to `pkg/namespace/v1alpha1`
This package will have structures, predicates & other functions !!!

Reply:
ok